adversarial_workflow/cli.py

```
@@ -2819,6 +2820,98 @@ def list_evaluators() -> int:
     return 0
 
 
+def check_citations(
+    file_path: str,
+    output_tasks: Optional[str] = None,
+    mark_inline: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)